Device: cuda
Dataset: cityscapes, Train set: 2975, Val set: 500
[!] Retrain
Traceback (most recent call last):
  File "main.py", line 567, in <module>
    main()
  File "main.py", line 515, in main
    loss.backward()
  File "/home/chenht/.conda/envs/epe/lib/python3.8/site-packages/torch/_tensor.py", line 363, in backward
    torch.autograd.backward(self, gradient, retain_graph, create_graph, inputs=inputs)
  File "/home/chenht/.conda/envs/epe/lib/python3.8/site-packages/torch/autograd/__init__.py", line 173, in backward
    Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
RuntimeError: CUDA out of memory. Tried to allocate 612.00 MiB (GPU 0; 23.70 GiB total capacity; 14.92 GiB already allocated; 580.62 MiB free; 15.17 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF